app/src/main/java/ca/marshallasch/veil/database/Migrations.java

```diff
@@ -15,7 +15,15 @@
      * In this version change the users table was added.
      * @param db the underlying database object
      */
-    static void v1ToV3(SQLiteDatabase db) {
+    /*static void upgradeV3(SQLiteDatabase db) {
```

Comment:
Leave in the old migration, just add in the new one

Reply:
resolved